aries-site/src/layouts/main/Layout.js

```diff
@@ -69,15 +63,10 @@ export const Layout = ({
                 }}
```

Comment:
Can the conditional on line 61 be removed? I think this should now be consistent across all pages.

Reply:
Yep! Good call.